Formula/mongodb-community@6.0.rb

```diff
@@ -0,0 +1,67 @@
+class MongodbCommunity < Formula
```

Comment:
This file is unchanged just renamed

Reply:
This needs to change to `MongodbCommunityAT60`